tests/functions/module_filter.test.ts

```diff
@@ -0,0 +1,71 @@
+import { setCellContent } from "../test_helpers/commands_helpers";
+import { getCellContent, getCellError } from "../test_helpers/getters_helpers";
+import { evaluateCell, getModelFromGrid, getRangeValuesAsMatrix } from "../test_helpers/helpers";
+
```

Comment:
will be good to test if UNIQUE make not difference between a raw value and a value inferred from formula
```
const grid = { A1: "3", A2: "=1+2", B1:"kikou", B2:"=CONCAT("ki","kou")}
setCellContent(model, "A1", "=UNIQUE(A1:B2)");
    expect(getRangeValuesAsMatrix(model, "D1:E3")).toEqual([
      ["3", "kikou"],
      ["", ""],
    ]);

expect(getRangeValuesAsMatrix(model, "A1:A2")).toEqual([["hey"], [""]]);
```